modules/hal_gpio/src/gpio.cpp

```diff
@@ -30,67 +30,68 @@
 
```

Comment:
This isn't refactored according to the google style guide guidelines... There shouldn't be indents after namespace